### ISSUE TITLE
[gatsby-theme-i18n] a wrong `md` locale node field is created when file extension is `.md`

### ISSUE THREAD

**Issue by twodft** [gatsbyjs/themes#157]
Hello, I'm using both `.md` and `.mdx` file extension for my blog posts. And while using `gatsby-theme-i18n`, 
I found it generate a wrong `locale` and `isDefault` node fields when my blog file extension is setting to `.md`,  the graphql result is like this,

```
        {
          "node": {
            "fields": {
              "isDefault": false,
              "locale": "md"
            }
          }
        },
```
after change my blog markdown file extension to `.mdx`, then I get the correct result.
```
{
          "node": {
            "fields": {
              "isDefault": true,
              "locale": "en"
            }
          }
        },
```

**Comment by olivier-lacroix** [comment]
@twodft , this would be fixed by https://github.com/gatsbyjs/themes/pull/130

if a maintainer merges it that is…

**Comment by twodft** [comment]
fixed by #130
